june/18/rajat_mehta.md

```diff
@@ -0,0 +1,6 @@
+# Yesterday 
+- learn about git and github 
+# Today
+- starting learning express.
+# Blockers
+- difficulty in contibuting in this project, i can contributing in frontend part but i registered uplift to learn backend.and while i see a backend issued i cant understand. so i decide first learn backend intermediate things then solve issues in /notalgify, Is this will be good way to go.Any suggestion?
```

Comment:
did you go through the sample code repository ?

Reply:
ya i saw the reference code. i understand few things but still not understand overall.